build/Dockerfiles/SIMP_EL8_Build.dockerfile

```diff
@@ -80,8 +97,8 @@ RUN runuser build_user -l -c "for i in {1..5}; do { gpg2 --keyserver  hkp://pool
 RUN runuser build_user -l -c "for i in {1..5}; do { gpg2 --keyserver  hkp://pool.sks-keyservers.net --recv-keys 7D2BAF1CF37B13E2069D6956105BD0E739499BDB || gpg2 --keyserver hkp://pgp.mit.edu --recv-keys 7D2BAF1CF37B13E2069D6956105BD0E739499BDB || gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 7D2BAF1CF37B13E2069D6956105BD0E739499BDB; } && break || sleep 1; done"
 #RUN runuser build_user -l -c "gpg2 --refresh-keys"
 RUN runuser build_user -l -c "curl -sSL https://raw.githubusercontent.com/rvm/rvm/stable/binscripts/rvm-installer -o rvm-installer && curl -sSL https://raw.githubusercontent.com/rvm/rvm/stable/binscripts/rvm-installer.asc -o rvm-installer.asc && gpg2 --verify rvm-installer.asc rvm-installer && bash rvm-installer"
-RUN runuser build_user -l -c "rvm install 2.4.10 --disable-binary"
-RUN runuser build_user -l -c "rvm use --default 2.4.10"
+RUN runuser build_user -l -c "rvm install 2.6"
```

Comment:
Why Ruby 2.6?

Reply:
Works with the least amount of fuss. 2.4 and 2.5 have some quirks and 2.7 is 3.0 prep.